images/plbase/python-requirements.txt

```diff
@@ -24,13 +24,13 @@ pytest-repeat==0.9.3
 pytest==8.3.3
 recommonmark==0.7.1
 regex==2024.9.11
-ruff==0.6.9
+ruff==0.7.1
```

Comment:
Lmao this is already out of date: https://github.com/astral-sh/ruff/releases/tag/0.7.2

Reply:
```suggestion
ruff==0.7.2
```